cloudinary-test-common/src/main/java/com/cloudinary/test/AbstractAccountApiTest.java

```diff
@@ -170,8 +174,10 @@ public void testGetUsers() throws Exception {
         returnedIds.add(((Map) users.get(0)).get("id").toString());
         returnedIds.add(((Map) users.get(1)).get("id").toString());
 
-        assertTrue("User1 id should be in the result set", returnedIds.contains(id1));
-        assertTrue("User2 id should be in the result set", returnedIds.contains(id2));
+        assertTrue("User1 id should be in the result set", returnedIds.contains(user1Id));
+        assertTrue("User2 id should be in the result set", returnedIds.contains(user2Id));
+        deleteUser(user1Id);
```

Comment:
putting `deleteUser` before assertions would provide a better chance for it to happen, like you perform request, then delete user, then assert results

Reply:
What has better chances to happen?

Reply:
you put `deleteUser(user1Id);` before all assertions

Reply:
I understood what you suggested, but couldn't figure out why is that better?

Reply:
it is better because because deleteUser(user1Id) will happen even if assertion fails, so we perform cleanup any way.
Not critical.